doc/api/stream.md

```diff
@@ -1706,8 +1704,14 @@ objects and passing appropriate methods as constructor options.
 const { Writable } = require('stream');
 
 const myWritable = new Writable({
+  construct(options, callback) {
```

Comment:
I'm open for other names. `setUp` sounds a bit clumsy. Maybe `init`. Though I do very much like `construct` since it goes well with `destroy`.

Reply:
Anyone else have feedback on this?

Reply:
How about `create`? Because `create` and `destroy` go well together too. Even more so than `construct` and `destroy`, because with `construct` I'd expect `destruct`.

Reply:
I'm fine with `create`. @mcollina what do you think?

Reply:
I'd vote for `init` instead of `create` considering we are not creating something new but actually initializing the already created stream using what was passed during the creation of the stream. Also, `init` and `destroy` go just fine together IMO. Though just a suggestion, feel free to ignore.

Reply:
+1 on init as the description is "initialize state and load resources" :)

Reply:
They are probably all fine. As long as the docs use consistent wording.

Although `construct` can be confused with constructing the JS object, it is also quite descriptive, seeing as constructing the object (rather than an additional external call) triggers `_construct()`. The fact that `construct` is async is slightly uncomfortable (due to lack of precedence) but with `create` or `init` you'd also have to explain that fact.

I suggested `create` for its symmetry with `destroy` and because I view Stream as an outer shell that creates, reads from, writes to, and then destroys a nut (aka "the implementation"). But I can see how `create` could be confusing too.

Lastly, I don't think `init` - which is overused and has many different meanings - really solves any of the concerns. It still has to be explained.

So, let's leave the bikeshed and stick with `construct`?  It sounds like no one has strong feelings about it.